tests/e2e/authcontroller_test.go

```diff
@@ -196,3 +195,37 @@ func (tc *AuthControllerTestCtx) validateAddingGroups() error {
 
 	return nil
 }
+
+func (tc *AuthControllerTestCtx) validateRemovingGroups() error {
+	expectedGroup := dashboard.GetAdminGroup()
+	if _, err := controllerutil.CreateOrUpdate(tc.ctx, tc.customClient, &tc.testAuthInstance, func() error {
+		tc.testAuthInstance.Spec.AdminGroups = []string{expectedGroup}
+		return nil
+	}); err != nil {
+		return errors.New("error removing groups from auth CR")
+	}
+
+	adminRolebinding := &rbacv1.RoleBinding{}
+	adminClusterRolebinding := &rbacv1.ClusterRoleBinding{}
+
+	if err := tc.testContext.customClient.Get(tc.ctx, types.NamespacedName{Namespace: tc.applicationsNamespace, Name: "admingroup-rolebinding"}, adminRolebinding); err != nil {
+		if len(adminRolebinding.Subjects) > 1 {
+			return fmt.Errorf("Expected 1 subject in adminRoleBinding found %v", len(adminRolebinding.Subjects))
+		}
+		if adminRolebinding.Subjects[1].Name != expectedGroup {
```

Comment:
Probably a copy&paste thing?
```suggestion
		if adminRolebinding.Subjects[0].Name != expectedGroup {
```

Reply:
:+1: